### PR TITLE
Cleanup test flows

### PR DESCRIPTION
**Pull request by yashgorana**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -266,11 +266,11 @@ allowlist_externals =
     chcp
 passenv=HOME, USER, AZURE_BLOB_STORAGE_KEY
 setenv =
-    HAGRID_FLAGS = {env:HAGRID_FLAGS:--tag=local --release=development --dev}
```
Comment (yashgorana) [inline tox.ini:269]: we don't need to deploy with `DEV_MODE=True` + use `docker-compose.dev.yaml`.

File: tox.ini [hunk 2]
```diff
@@ -680,49 +666,23 @@ commands =
...
-
-    ; container workload
-    ; bash -c 'if [[ "$PYTEST_MODULES" == *"container_workload"* ]]; then \
-    ;     echo "Starting Container Workload test"; date; \
-    ;     pytest tests/integration -m container_workload -p no:randomly --co; \
-    ;     pytest tests/integration -m container_workload -vvvv -p no:randomly -p no:benchmark -o log_cli=True --capture=no; \
-    ;     return=$?; \
-    ;     echo "Finished container workload"; date; \
-    ;     exit $return; \
-    ; fi'
```
Comment (yashgorana) [inline tox.ini:716]: @rasswanth-s container workloads for k8s has been disabled. please confirm if this needs to be re-enabled.

Comment (rasswanth-s) [reply tox.ini:716]: The container workloads has been disabled for some time in k8s I think.
This is a tech debt , we could re-enable it later.

**Comment by yashgorana** [comment]
I have to re-prioritize to focus on critical fixes. Closing this PR + dropping the 0.8.7 label. Will re-open soon.

**Pull request closed without merging**